src/layer/one-to-n-relations/index.md

```diff
@@ -7,53 +7,72 @@ You can clone these projects to take a closer look on 1-N relations:
    - Adding multiple photos to a single feature: <MerginMapsProject id="documentation/forms_multiple_photos" />
 :::
 
-It is often the case that you have a set of spatial features and you want to record some parameters every now and then. For example, there is a GIS layer representing the manholes and the surveyors carry out regular inspections of the manholes. Instead of duplicating the manhole layer and recording each inspection, you can create a non-spatial table and store each inspection as a new row.
+It is often the case that you have a set of spatial features and you want to record their status every now and then. For example, there is a GIS layer representing the manholes and the surveyors carry out regular inspections of the manholes using <MainPlatformNameLink />. Instead of duplicating the manhole layer and recording each inspection as a new feature, inspections can be recorded in a non-spatial table that is linked to the spatial layer. This way, multiple records can be linked to one feature.
 
-The image below shows the manholes locations:
+The image below shows the manhole locations and a form with listed inspections in <MobileAppName />.
 
-![Manhole locations](./input_forms_many-relations_concept1.png)
+![one feature with multiple inspection records](./input-1-n.jpg "One point - multiple inpections")
 
-The manhole point layer has the following attribute table:
+The manhole point layer has the following attribute table: 
 
-![Manhole locations](./input_forms_many-relations_concept2.png)
+| fid | Manhole | Manhole UUID |
+|:---:|:---:|:---:|
+| 1 | 1 | `{70c59616-492e-4757-aa9a-ee61b207ce94}` |
+| 2 | 2 | `{be01b98f-3585-49d4-be74-4cf3530a2989}` |
+| 3 | 3 | `{03178264-0070-45c8-a981-b2474627d7e0}` |
 
-In a separate [non-spatial table](../working_with_nonspatial_data/), we can record the inspections:
+This layer contains only information about the manholes. `Manhole UUID` values are generated using [`uuid()` function as a default value](../attach-multiple-photos-to-features/) when a feature is created. This ensures that these values are **unique** even when multiple surveyors capture new features at the same time. This field will be used to link inspections and manholes.
 
-![Manhole locations](./input_forms_many-relations_concept3.png)
+:::danger
+**Why UUIDs?** FIDs can be changed during [synchronisation](../../manage/synchronisation/#synchronisation). As a result, records can end up being linked to wrong features. 
 
-In QGIS, we are using the **Manhole ID** from the point layer and the inspection table to create a 1-N relation between the two tables.
+On the other hand, <QGISHelp ver="latest" link="user_manual/expressions/functions_list.html#uuid" text="UUID" /> (Universally Unique Identifier) is generated to be unique and will not be changed when synced. Therefore, we recommend always using UUIDs as primary keys.
```

Comment:
"Therefore, we recommend always using UUIDs as primary keys." -> we do not use UUIDs as primary keys, only as foreign keys for inspections, no?